mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -467,6 +470,96 @@ def test_spectral_connectivity(method, mode):
     assert out_lens[0] == 10
 
 
+# Fourier coeffs in Spectrum objects added in MNE v1.8.0
+@pytest.mark.skipif(
+    Version(mne.__version__) <= Version("1.7.1"), reason="Requires MNE v1.8.0 or higher"
+)
```

Comment:
Would ideally have checked version against 1.8, but dev versions are considered earlier than official releases. Will need to change this when MNE 1.8 is released.

Reply:
In `mne.utils.check_version` actually we treat dev versions differently, if you use that then it should do what you want

Reply:
```
>>> import mne
>>> mne.__version__
'1.8.0.dev123+g790be2d0f'
>>> mne.utils.check_version("mne", "1.8")
True
```

Reply:
Have now added a proper reminder in code to do this

Reply:
No I mean you can do it now, in this PR. If you do
```
pytest.mark.skipif(not check_version("mne", "1.8"), ...)
```
it will work now and once 1.8 is released just fine, no need for any "change this later" comment

Reply:
Sorry, I didn't refresh the page before adding that reminder comment, will switch to your solution.

Reply:
Have switched to that now